### DISCUSSION TITLE
TransferProcessManagerImpl - Code duplication

### DISCUSSION THREAD

**Discussion by klumbe** [eclipse-edc/Connector#4696] [General]
Since I'm afraid to open a bug for such a small thing (which also won't change the behavior):

There is an unnecessary second call for creating the `dataDestination` for a `TransferProcess` in the `TransferProcessManagerImpl`:
1. [L147](https://github.com/eclipse-edc/Connector/blob/4eca6753b6de63a6664caa850f3b0d7dbb9f721d/core/control-plane/control-plane-transfer/src/main/java/org/eclipse/edc/connector/controlplane/transfer/process/TransferProcessManagerImpl.java#L147)
2. [L151](https://github.com/eclipse-edc/Connector/blob/4eca6753b6de63a6664caa850f3b0d7dbb9f721d/core/control-plane/control-plane-transfer/src/main/java/org/eclipse/edc/connector/controlplane/transfer/process/TransferProcessManagerImpl.java#L151)

Let me know if I should create an issue ;-)

**Comment by jimmarino** [comment]
@klumbe That is definitely a typo-bug. A fix would be appreciated!

**Reply by klumbe** [reply]
Would have already provided one, if there weren't the requirement for a Google-Account to sign the CLA.
